src/pages/ProbeHybridisationQC.tsx

```diff
@@ -0,0 +1,391 @@
+                  <Heading level={2}>SGP Number</Heading>
+                  <p>Select an SGP number to associate with this operation.</p>
+                  <div className="mt-4 md:w-1/2">
+                    <WorkNumberSelect
+                      name="workNumber"
+                      onWorkNumberChange={(workNumber) => {
+                        setFieldValue('workNumber', workNumber);
+                      }}
+                    />
+                  </div>
+                </div>
+                <div className="mt-8 space-y-2">
+                  <Heading level={2}>Slides</Heading>
+                  <p>Please scan a slide</p>
+                  <LabwareScanner onAdd={onAddLabware} onRemove={onRemoveLabware}>
+                    {({ labwares, removeLabware }) =>
+                      labwares.map((labware) => (
+                        <Panel key={labware.barcode}>
+                          <div className="flex flex-row items-center justify-end">
+                            {
+                              <RemoveButton
+                                data-testid={'remove'}
+                                onClick={() => {
+                                  removeLabwareFromForm(removeLabware, labware.barcode, values);
+                                }}
+                              />
+                            }
+                          </div>
+                          <div className="flex flex-row">
+                            <div className="flex flex-col w-full" data-testid={'labware'}>
+                              <Labware labware={labware} name={labware.labwareType.name} />
+                            </div>
+                            <div className="flex flex-col w-full bg-gray-100">
+                              <div className="p-4">
+                                <FormikInput
+                                  label={'Completion Time'}
+                                  data-testid={'completionDateTime'}
+                                  type="datetime-local"
+                                  name={`labwares[${labware.barcode}].completionDateTime`}
+                                  max={currentDateTime}
+                                  min={currentDateTime + 'T00:00'}
+                                  onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                                    setFieldValue(`labwares.${labware.barcode}.completionDateTime`, e.target.value);
+                                  }}
+                                  value={values.labwares[labware.barcode]?.completionDateTime || ''}
+                                  className="w-1/2"
+                                  validate={validateCompletionDateTime}
+                                />
+                                <MutedText className="pl-2">
+                                  If not manually selected, the current timestamp will be applied automatically
+                                </MutedText>
```

Comment:
Please make use of formik error handling capabilities, by including this field in validation schema , so we can avoid using MutedText and validateCompletionDateTime

Reply:
hmmm I am not sure how to include this in the formik errors as the request is still valid, when the completion time is not been selected

Reply:
I would think of it as having a default value which is the current time displayed in the field when it opens, and the validation can be based on that time which is the default